packages/protocol/src/action/client/subscription/ClientSubscribe.ts

```diff
@@ -5,4 +5,9 @@ export interface ClientSubscribe extends Subscribe {
      * If true the subscription is virtualized and the underlying subscription is reestablished when lost.
      */
     sustain?: boolean;
+
+    /**
+     * If true, the request will be queued over all peers of the node
+     */
+    queued?: boolean;
```

Comment:
It feels wrong to push this decision to the subscriber.  I could see maybe having an override here for very advanced cases, but shouldn't the protocol stack be deciding this?  I was thinking something like `clientNetworkingLimits: { queueSize: 100, threadQueueSize: 5 }` in `NetworkServer#state`

Which I realize isn't in scope for this PR but wondering about sticking it here...  Maybe could add @deprecated if for back-compat?

Reply:
I agree that we should change this. I added a TODO and also deprecate. For now I need it that the Controller works "compatible".